humility-idol/src/lib.rs

```diff
@@ -285,6 +285,36 @@ fn lookup<'a>(
     }
 }
 
+// Convert string representation of array from the hiffy `--argument` list
```

Comment:
Style nit: since this is a docstring, it should have `///` (since some editors will highlight docstrings differently and show them through LSP hints).

If you want to be Very Strict, there should be a one-line summary followed by details after a blank line, per [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text).

Reply:
Should be addressed in my last push. Hopefully the text makes sense.

Reply:
Looks great, thanks!